.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
           - types-dataclasses >= 0.1.3
           - types-toml >= 0.1.1
           - types-typed-ast >= 1.4.1
-          - types-click >= 7.1.2
+          - click >= 7.1.2
```

Comment:
The linked Click issue was milestoned to version 8. Should this dependency be upgraded then?